utils.py

```diff
@@ -0,0 +1,14 @@
+import spaceship
+
+def get_enemy_size(enemy_type):
+    match enemy_type:
+        case 'enemy1': 
+            return 25
+        case 'enemy2':
+            return 45
+        case 'boss3':
+            return spaceship.WIDTH * 4 // 2
+        case 'boss4':
+            return spaceship.WIDTH * 5 // 2
+        case 'boss5':
+            return spaceship.WIDTH * 6 // 4
```

Comment:
The question of where to go (in the code file) to adjust enemy sizing was also a little bit scattered. 

I concentrated it in one place. Since it's not quite related to the direct creation of the enemy, I put it in a separate file, `utils.py`. (You may disagree and put it elsewhere - and that's OK! It's your code!)